src/app/role-admin/employee-list/employee-list.component.ts

```diff
@@ -0,0 +1,54 @@
+import { Component, OnInit } from '@angular/core';
+import {  Router, TitleStrategy } from '@angular/router';
+import { ListServiceService } from '../../list-service.service';
+
+
+
+@Component({
+  selector: 'app-employee-list',
+  templateUrl: './employee-list.component.html',
+  styleUrls: ['./employee-list.component.css']
+})
+export class EmployeeListComponent implements OnInit{
+
+  emplist: any = null;
+  addform: any = false;
+  empdetailform: any = true;
+  laptop: any;
+//  public sm=this.emplist.name.split("")
+  
+
+  constructor(private listservice: ListServiceService, private router:Router) { }
+
+
+  ngOnInit(): void {
+    this.getlist()
+  
+ 
+  }
+  getlist() {
+     this.listservice.getlist().subscribe((response) => {
```

Comment:
when ever you use a subscription. on destroy of the component unsubscribe the subscription. to avoid multiple subscriptions